packages/block-editor/src/components/block-preview/index.js

```diff
@@ -98,7 +112,7 @@ function ScaledBlockPreview( { blocks, viewportWidth } ) {
 	);
 }
 
-export function BlockPreview( { blocks, viewportWidth = 700, settings } ) {
+export function BlockPreview( { blocks, viewportWidth = 700, settings, __experimentalOnReady = noop, __experimentalDelay = 100 } ) {
```

Comment:
The delay we have is for the addition of scaling right? Maybe we can call the prop __experimentalScallingDelay to be more specific I guess ideally the name would refer to what is being delayed.
I think we should also document this prop together with __experimentalOnReady.

Reply:
with only one `l` ;-)